common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -255,6 +255,59 @@ color_lists = {
 			125 = "black"
 			250 = "black_empire_dark_red"
 		}
+		
+		###### Warcraft Culture Specific CoAs ######
+		
+		#### Humans ###
+		
+		special_selection = { 
+			trigger = { azerothian_coa_trigger = yes }
+			125 = "gold_alliance"
+			250 = "blue_alliance"
+		}
+		special_selection = {
+			trigger = { lordaeronian_coa_trigger = yes }
+			125 = "blue_alliance" #Blue Insignia
+			125 = "gold_alliance"
+			125 = "grey_dark"
+			250 = "silver_lordaeron"
+		}	
+		special_selection = {
+			trigger = { arathorian_coa_trigger = yes }
+			75  = "gold_alliance" #League of Arathor
+			125 = "silver_lordaeron"
+			250 = "red_strom"
+		}
+		special_selection = {
+			trigger = { alteraci_coa_trigger = yes }
+			125 = "brown_alterac"
+			125 = "black"
+			250 = "orange"
+		}
+		special_selection = {
+			trigger = { gilnean_coa_trigger = yes }
+			125 = "gold_alliance"
+			125 = "rose_gilneas"
+			125 = "grey_gilneas"
+			250 = "black"
+		}
+		special_selection = {
+			trigger = { tirassian_coa_trigger = yes }
+			125 = "gold_alliance"
+			125 = "blue_alliance" #Theramore
+			250 = "green_tirassian"
```

Comment:
Also, there are other colors.
https://wowpedia.fandom.com/wiki/House_Waycrest
https://wowpedia.fandom.com/wiki/House_Stormsong
https://wowpedia.fandom.com/wiki/House_Ashvane

I suggest making them less common or tying to regions (houses from the Stormsong Valley having blue CoAs).